tests/unit/test_implementation.py

```diff
@@ -12,17 +11,4 @@ def test_fails_when_missing_results(default_config, mocker):
         config=default_config,
         step=Step("step_1"),
     )
-    mocker.patch("linker.runner.run_container", return_value=None)
-    mocker.patch("linker.runner.run_with_singularity", return_value=None)
-    with pytest.raises(
-        RuntimeError,
-        match="No results found for pipeline step ID .* in results directory '.*'",
-    ):
-        implementation.run(
-            session=None,
-            runner=run_container,
-            step_id="step_1",
-            input_data=[],
-            results_dir=Path("some-path"),
-            diagnostics_dir=Path("some-path"),
-        )
+    pass
```

Comment:
Snakemake does this now, not the implementation, but I kind of didn't want to remove it entirely to remind myself about it.